packages/interval/index.ts

```diff
@@ -1,3 +1,38 @@
+type IntervalName = string;
+type IntervalQuality = string;
+type IntervalAlteration = number;
+type IntervalStep = number;
+type IntervalDirection = 1 | -1;
+type IntervalSimplifiedNumber = number;
+type IntervalType = "P" | "M";
+type IntervalChroma = number;
```

Comment:
From docs: "The chroma is a number between 0 and 7 (...)" ... does have any impact on the type?